package com.cmc.rest.test;

import org.apache.logging.log4j.Logger;

import com.cmc.rest.commons.ValidationException;
import com.cmc.rest.entidades.Persona;
import com.cmc.rest.servicios.ServicioPersonas;

public class TestActualizar {

	public static void main(String[] args) {
		Persona persona=new Persona("1231231","Cristi","Mora");
		System.out.println(persona.toString());
		try{
		Persona n=ServicioPersonas.actualizar(persona);
		}catch(ValidationException e){
			e.printStackTrace();
			logger.error("Se a producido un error en el programa", e);
		}
		System.out.println(n.toString());
		

	}

}
